specs/File Classification Spec.md

```diff
@@ -0,0 +1,95 @@
+
+## Requirements
+
+### Functional Requirements
+
+**1. Initial UX/UI**
+   
+   Users must open File Explorer and select a group of files by checking each file or using Shift + directional keys.  They must then use a shortcut to open the File Classification utility, which opens in the center of the screen and provides options that require input from the user.
+
+   **For a visual example of the UX see Figure 1*
+
+   **For limitations in user input see 2.4*
+
+**2. Rename File Options**   
+
+   2.1. Simple File Rename
+   - Renames all user-selected files after receiving a label, which is typed by the user to be the files' new name.
+     - Example: If the user types that the label is "C" then selected files named "A" and "B" would be renamed to "C(1)" and "C(2)"
+
+   2.2. Folder and File Rename
+   - Automates folder creation and movement of user-selected files into the new folder.
+   - The user must type in a label, which will serve as the name of the folder and renames the files.
+     - Example: If the user types that the label is "Hello" then a folder will be created and named "Hello". The selected files will be moved to the folder and renamed "Hello(1)"...
+
+   2.3. Replace by Characters
+   - Identifies a group of characters, which are typed by the user, in the names of user-selected files and replaces that group of characters with characters input by the user.
+   - Every instance of the characters found in each selected file's name will be replaced.
+     - Example: If the user types that "e" should be replaced with "o" then a selected file named "bleed" would be changed to "blood"
+
+   2.4. Shared Functionality
+   
+   - **Labels**
+     - Used as the name for the new folders created.  
+     - Used to rename files by completely overwriting the files' current names with the custom 'Label' typed by the user and each file receives a counting number.
+   - **Confirmation**  
+     - After the user selects an option and fills in the required text boxes, they must click 'Apply' for the utility to make the changes, or they can click 'Cancel' which closes the utility without making changes.
+   - **Input Limitations**
+     - Only a single option can be applied at a time, so if the user begins to type text for one option, the others options should block input (*text area turns grey?*) unless input is removed from the original option.
+     - If a space alone is inputted into any of the options, no changes are made when apply is clicked.
+- **Shortcut Accessibility**
+  - Perhaps Ctrl + Tab, which does not have a function within File Explorer.
+  - Alternatives include : Alt + C , Ctrl + Alt + F
+
+Figure 1: Initial UX/UI Visual
+![Figure 1-1](https://raw.github.com/indierawk2k2/PowerToys-1/images/PT%20Images/File%20Classification%20Design%20Blurred.png "FC")
+
+### Measure Requirements
+
+- A score of an average of 4 stars on a new Consumer Design Satisfaction Survey
+- 100 Github comments
```

Comment:
Our thought process was that comments and pull requests generally would be used as a means of gaging general interest more than anything positive or negative.  We're also considering looking at forks and stars in this manner.  This is more-so referring to consumer interest and interaction through the design process rather than a reaction to the final product.

Reply:
Ah I see, so more of taking a look at how successful you're being while developing it?
I'd recommend that this section focuses on the final result that you're looking to achieve with this product, and I think pull requests and Github issues aren't a great measure of success for a final product.